yarn.lock

```diff
@@ -5134,29 +5210,29 @@
     eslint-visitor-keys "^3.0.0"
 
 "@utrecht/component-library-css@^1.0.0-alpha.343":
-  version "1.0.0-alpha.343"
-  resolved "https://registry.yarnpkg.com/@utrecht/component-library-css/-/component-library-css-1.0.0-alpha.343.tgz#87fdb158b63804dc6f0f1e03875fc0bb6531c9db"
-  integrity sha512-a2+b5ewCN/kbVQYHuSORB3vYwaa6o0Oj8zNd17z+9IJ9tQE2VTOeElaVXBQ6Y1EnkT+8i3rTVn3H2ckHifceaw==
+  version "1.0.0-alpha.365"
```

Comment:
![image](https://user-images.githubusercontent.com/325643/199808025-9489268b-a8bf-4381-98ed-425551d266f4.png)
This is what it looks like in the [Utrecht storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-alternate-lang-nav--button-group). I think for visual a11y, there should be a visual separator between the buttons.

Reply:
I think you could use the following, similar to the alternate lang link approach:

```jsx
<Button inline>EN</Button><span aria-hidden="true">{' | '}</span><Button inline>NL</Button>
```